torch/csrc/jit/passes/quantization/insert_observers.cpp

```diff
@@ -124,11 +124,24 @@ class ModuleCloneHelper {
     size_t N = type->numAttributes();
     for (size_t i = 0; i < N; ++i) {
       IValue s = module._ivalue()->getSlot(i);
-      if (type->getAttribute(i)->is_module()) {
```

Comment:
Extremely suspicious that we have two highly similar versions of cloning :/, but that is not related to this PR